Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1812,7 +1813,11 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const minWidth = LayoutUtility.GetOuterWidth(reference);
         const maxWidth = Math.min (isNaN (options.maxWidth) ? LayoutUtility.GetOuterWidth(reference) : options.maxWidth, availableWidth);
         const requiredWidth = contentWidth + 30;
-        const elementWidth = Math.max (Math.min (requiredWidth, maxWidth), minWidth);
+
+        let elementWidth: number | string;
```

Comment:
This code change also affects the result-list of the autocmplete (same calculation function is used). We need to test the expand/shrink-behavior of both the infobox and the resultlist. Do this tomorrow as a pair?

Reply:
With @a-ctor's change this has become redundant. We should more likely talk about whether this is the intended way the message should be displayed, as it now does not follow the design specified in the issue.